src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json

```diff
@@ -1080,72 +1080,72 @@
       "name": "media",
       "Files": [
         {
-          "Path": "about/1.jpg",
-          "Base64": "[file:base64('../wwwroot/img/about/1.jpg')]"
+          "TargetPath": "about/1.jpg",
+          "SourcePath": "Areas/TheAgencyTheme/wwwroot/img/about/1.jpg"
```

Comment:
can't we use `../` like before?

Reply:
Well, the file provider that is configured is the CompositeFileProvider with the ModuleEmbeddedFileProvider and PhysicalFileProvider - and only the first one is ever invoked. I hesitated to enhance any of this because the only solution I could think of was to add to ModuleEmbeddedFileProvider a brute-force approach to check every module whether that relative path applied.

Reply:
Never mind ... I see the BasePath property on the recipe descriptor ... looking through the code for the proper relative path resolution code.

Reply:
@sebastienros I can either clone the GetRelativeFile() code from FileScriptScope here but I'd like to see a more centralized and reusable implementation and refactor FileScriptScope to use that implementation. Any suggestions on where I can put this reusable implementation and what should I name it?

Reply:
@sebastienros I can either clone the GetRelativeFile() code from FileScriptScope here but I'd like to see a more centralized and reusable implementation and refactor FileScriptScope to use that implementation. Any suggestions on where I can put this reusable implementation and what should I name it?

Reply:
@sebastienros I cloned the GetRelativeFile() code from FileScriptScope into MediaStep. You can decide whether you want to leave it in the MediaStep or you want to move it behind an interface that FileScriptScope can also reuse. Please review and provide feedback.